[INFO   ] Logger: Record log in /storage/emulated/0/Progs/NeroProgs/Kivy/Lista de exercícios python/11-calculando valores diversos/.kivy/logs/kivy_20-08-08_6.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.8.3 (default, Jun 15 2020, 02:19:10) 
[GCC 9.3.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/bin/python3"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_gif (img_pil, img_ffpyplayer ignored)
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/Progs/NeroProgs/Kivy/Lista de exercícios python/11-calculando valores diversos/processando.py", line 6, in <module>
[WARNING] stderr:     Calcula().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/app.py", line 828, in run
[WARNING] stderr:     self.load_kv(filename=self.kv_file)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/app.py", line 599, in load_kv
[WARNING] stderr:     root = Builder.load_file(rfilename)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/builder.py", line 301, in load_file
[WARNING] stderr:     return self.load_string(data, **kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/builder.py", line 368, in load_string
[WARNING] stderr:     parser = Parser(content=string, filename=fn)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/parser.py", line 401, in __init__
[WARNING] stderr:     self.parse(content)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/parser.py", line 514, in parse
[WARNING] stderr:     rule.precompile()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/parser.py", line 261, in precompile
[WARNING] stderr:     x.precompile()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/parser.py", line 259, in precompile
[WARNING] stderr:     x.precompile()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/parser.py", line 184, in precompile
[WARNING] stderr:     self.co_value = compile(value, self.ctx.filename or '<string>', mode)
[WARNING] stderr:   File "/storage/emulated/0/Progs/NeroProgs/Kivy/Lista de exercícios python/11-calculando valores diversos/calcula.kv", line 43
[WARNING] stderr:     lab.text =
[WARNING] stderr:              ^
[WARNING] stderr: SyntaxError: invalid syntax
